apps/sms/index.html

```diff
@@ -115,6 +116,7 @@
     <script defer src="views/shared/js/activity_handler.js"></script>
     <script defer src="views/shared/js/contact_renderer.js"></script>
     <script defer src="views/conversation/js/information.js"></script>
+    <script defer src="/js/localization_helper.js"></script>
```

Comment:
Ha, yes, thanks! It basically wasn't required I just wanted to add it for the consistency.